imagevector/images.yaml

```diff
@@ -296,7 +296,7 @@ images:
 - name: coredns
   sourceRepository: github.com/coredns/coredns
   repository: registry.k8s.io/coredns/coredns
-  tag: "v1.10.1"
+  tag: "v1.11.1"
```

Comment:
Can you also update the coredns version used in local setup?

Ref https://github.com/gardener/gardener/blob/328f202a312de139bb9ae80378c3641c254850c3/charts/gardener/provider-local/templates/coredns/deployment.yaml#L32